uber/models.py

```diff
@@ -856,6 +861,7 @@ def model_mixin(cls, model):
 
 
 class Group(MagModel, TakesPaymentMixin):
+    search_key    = Column(UUID, default=lambda: str(uuid4()))
```

Comment:
Yes, it searches a bunch of different fields with the original full search term if the stuff after the EVENT_QR_ID turns out not to be a UUID.